docs/topics/Gateway.md

```diff
@@ -781,6 +790,29 @@ A user's presence is their current state on a guild. This event is sent when a u
 >info
 >The streaming type currently only supports Twitch. Only `https://twitch.tv/` urls will work.
 
+###### Game Timestamps
+
+| Field | Type | Description |
+|-------|------|-------------|
+| start? | int | Epoch millis of start |
+| end? | int | Epoch millis of end |
+
+###### Game Party
+
+| Field | Type | Description |
+|-------|------|-------------|
+| id? | string | The ID, relevant for joining the party |
+| size? | array of two integers | First element is the current size, second the max `[2, 5]` |
```

Comment:
should bring this in-line with how we represent `shard`, which is also a tuple

type:
`array of two integers (current_size, max_size)`

description:
`used to show the party's current and maximum size`